_includes/scenarios/integrate/request-and-response-introduction.md

```diff
@@ -0,0 +1,107 @@
+
+Since each Response is associated with a specific Request, the corresponding `requestId` is indicated in the Response.
+Its `result` property declares whether the Request was accepted or rejected.
+Also, each ResponseItem has a `result` property itself, whose value depends on the kind of ResponseItem and can either be `Accepted`, `Rejected` or `Error`.
+In addition, RejectResponseItems and ErrorResponseItems may have a `code` and a `message` specified, providing more details about the rejection or error.
+The ResponseWrapper holds additional information about the Request, namely its ID, the `requestSourceType`, i.e. if the Request was transmitted via a [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate) or a [Message]({% link _docs_integrate/data-model-overview.md %}#message), and the `requestSourceReference`, containing the respective ID.
+
+## Examples
+
+### Required and optional RequestItems
+
+An organization has a Relationship with a customer and needs their consent to the privacy terms.
+Additionally, the organization would like to know, if the customer is interested in optionally receiving a newsletter.
+Thus, they send a Request, containing two ConsentRequestItems.
+On the one hand, the RequestItem regarding consent to the Privacy Terms has the `mustBeAccepted` flag enabled, indicating that the Request can't be accepted without accepting this item.
+On the other hand, the RequestItem regarding the Newsletter has the `mustBeAccepted` flag disabled.
+Hence, the customer can freely choose whether or not they would like to give their consent to it.
+Let's consider the case the Request and therefore the privacy terms are accepted, but the consent to the newsletter is denied.
+The resulting Request-Response flow is depicted in the following graphic.
+For simplicity some properties are omitted.
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/8989a397-d7e8-4c3c-b447-0d9043da8ceb" id="7CvNv9u5~~Sl"></iframe></div>
+
+### Working with RequestItemGroups
+
+At a job fair a company wants to offer a convenient way to get in touch with interested jobseekers.
+For this, they provide a QR-code, linking to a RelationshipTemplate.
+In its `content.onNewRelationship` property it holds a Request with two RequestItemGroups.
+One of them contains Attributes the company shares with the peer, e.g. the company name.
+The other contains Attributes it would like to query from the peer.
+In this example they are the given and surname and optionally an e-mail address, following the [Integration example]({% link _docs_integrate/integration-example.md %}).
+Now, an interested person can scan the QR-code, provide their information and send their Response inside a RelationshipCreationChangeRequest (note that this is not the kind of Request discussed on this page).
```

Comment:
Careful here, due to the underlying RelationshipCreationChangeRequest a LocalRequest is in fact created (and already responded to) and can be used.
Should we consider removing the note on RelationshipCreationChangeRequest altogether here?